delta-api/src/main/java/io/cdap/delta/api/Configurer.java

```diff
@@ -18,9 +18,19 @@
 
 import io.cdap.cdap.api.plugin.PluginConfigurer;
 
+import javax.annotation.Nullable;
+
 /**
  * Configures a data transfer.
  */
 public interface Configurer extends PluginConfigurer {
 
+  @Nullable
+  String getNamespace();
+
+  @Nullable
+  String getName();
+
+  @Nullable
+  Long getGeneration();
```

Comment:
Do we need the namespace generation? This might be confusing to have since it's different than the pipeline id generation.

If we need it, I think it would be better to move Namespace to delta-api and make getNamespace() return that class, that way it's more clear that it's the namespace generation.

Reply:
I was thinking generation was one of the identifier of namespace. If that's not the case, I don't think we need it.